src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__month month month--length--31">
+      <div class="month__day month__day--start-day--sun month__day--number-day--1">
+      </div>
+      <div class="month__day month__day--number-day--2"></div>
```

Comment:
better delete classes `month__day--number-day--N` and use `:nth-child()` to styling.
```suggestion
      <div class="month__day"></div>
```

Reply:
done